main/params.go

```diff
@@ -419,138 +477,216 @@ func init() {
+		return fmt.Errorf("failed to set fd limit correctly due to: %w", err)
 	}
 
+	// Network Parameters
 	if networkID != constants.MainnetID && networkID != constants.FujiID {
-		Config.TxFee = *txFee
-		Config.CreationTxFee = *creationTxFee
-		Config.UptimeRequirement = *uptimeRequirement
-		Config.UptimeRequirement = *uptimeRequirement
 
-		if *minValidatorStake > *maxValidatorStake {
-			errs.Add(errors.New("minimum validator stake can't be greater than maximum validator stake"))
+		txFee := v.GetUint64(txFeeKey)
+		creationTxFee := v.GetUint64(creationTxFeeKey)
+		uptimeRequirement := v.GetFloat64(uptimeRequirementKey)
+		Config.TxFee = txFee
+		Config.CreationTxFee = creationTxFee
+		Config.UptimeRequirement = uptimeRequirement
+
+		minValidatorStake := v.GetUint64(minValidatorStakeKey)
+		maxValidatorStake := v.GetUint64(maxValidatorStakeKey)
+		minDelegatorStake := v.GetUint64(minDelegatorStakeKey)
+		minDelegationFee := v.GetUint64(minDelegatorFeeKey)
+		if minValidatorStake > maxValidatorStake {
+			return errors.New("minimum validator stake can't be greater than maximum validator stake")
 		}
 
-		Config.MinValidatorStake = *minValidatorStake
-		Config.MaxValidatorStake = *maxValidatorStake
-		Config.MinDelegatorStake = *minDelegatorStake
+		Config.MinValidatorStake = minValidatorStake
+		Config.MaxValidatorStake = maxValidatorStake
+		Config.MinDelegatorStake = minDelegatorStake
 
-		if *minDelegationFee > 1000000 {
-			errs.Add(errors.New("delegation fee must be in the range [0, 1000000]"))
+		if minDelegationFee > 1000000 {
+			return errors.New("delegation fee must be in the range [0, 1000000]")
 		}
-		Config.MinDelegationFee = uint32(*minDelegationFee)
+		Config.MinDelegationFee = uint32(minDelegationFee)
 
 		if Config.MinStakeDuration == 0 {
-			errs.Add(errors.New("min stake duration can't be zero"))
+			return errors.New("min stake duration can't be zero")
 		}
 		if Config.MaxStakeDuration < Config.MinStakeDuration {
-			errs.Add(errors.New("max stake duration can't be less than min stake duration"))
+			return errors.New("max stake duration can't be less than min stake duration")
 		}
 		if Config.StakeMintingPeriod < Config.MaxStakeDuration {
-			errs.Add(errors.New("stake minting period can't be less than max stake duration"))
+			return errors.New("stake minting period can't be less than max stake duration")
 		}
 	} else {
 		Config.Params = *genesis.GetParams(networkID)
 	}
+
+	// Consensus Parameters
+	Config.ConsensusParams.K = v.GetInt(snowSampleSizeKey)
+	Config.ConsensusParams.Alpha = v.GetInt(snowQuorumSizeKey)
+	Config.ConsensusParams.BetaVirtuous = v.GetInt(snowVirtuousCommitThresholdKey)
+	Config.ConsensusParams.BetaRogue = v.GetInt(snowRogueCommitThresholdKey)
+	Config.ConsensusParams.Parents = v.GetInt(snowAvalancheNumParentsKey)
+	Config.ConsensusParams.BatchSize = v.GetInt(snowAvalancheBatchSizeKey)
+	Config.ConsensusParams.ConcurrentRepolls = v.GetInt(snowConcurrentRepollsKey)
+
```

Comment:
Can we remove this empty line

Reply:
This is a separator between the Config.ConsensusParams attributes and consensus parameters specified directly on Config.